woocommerce/woocommerce-gutenberg-products-block
# Show the phone number field in the billing section when shipping is disabled in settings

Moves the rendering of address fields to a single file so that PhoneNumber (new component) can be rendered in _either_ billing or shipping, depending on visibility, without duplication.

I was going to update context to handle this (hence why I cleaned up some missing docblocks) however, it made more sense to handle this within the address-step component and simply pass around the attributes as before.

Fixes #3231
Fixes #3346 

### How to test the changes in this Pull Request:

Ensure the phone number field is enabled in block settings, then test 2 cases:

1. With shipping enabled, ensure phone number displays beneath the shipping step
2. With shipping disabled (you can delete all rates and zones to edit general settings), ensure phone number displays beneath the billing step.

In both scenarios the values should change and be saved as billing information.

### Changelog

> Show the phone number field in the billing section when shipping is disabled in settings
